Iterable/iterable-web-sdk
# Embedded text wrap

## JIRA Ticket

* [MOB-7514](https://iterable.atlassian.net/browse/MOB-7514)

## Description

> Modify the style to resolve text wrapping issue.

## Test Steps

> Start by launching the sample app, then select the 'embedded Msgs' option. Enter your userId/email in the designated field. Afterward, choose the option you wish to test and see the result.

[MOB-7514]: https://iterable.atlassian.net/browse/MOB-7514?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
